uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/components/leave-request-popup-details-tab.component.js

```diff
@@ -138,23 +138,28 @@ define([
     /**
      * Calculate change in balance, it updates local balance variables.
      *
-     * @return {Promise} empty promise if all required params are not set otherwise promise from server
+     * @param  {Boolean} toGetOriginalBreakdown
+     * @return {Promise} empty promise if all required params are not set
+     *   otherwise promise from server
      */
-    function calculateBalanceChange () {
+    function calculateBalanceChange (toGetOriginalBreakdown) {
```

Comment:
This function now is doing two different things (the boolean value is a clear indicator of this issue). It would be better to have two separate functions with no arguments: `getBalanceChangeBreakdown` and `calculateBalanceChangeBreakdown` (or similar names).